ReactiveXComponent/RabbitMq/RabbitMqPublisher.cs

```diff
@@ -73,6 +73,16 @@ public void GetSnapshotAsync(string stateMachine, Action<List<MessageEventArgs>>
             _rabbitMqSnapshotManager.GetSnapshotAsync(stateMachine, onSnapshotReceived);
         }
 
+        public List<string> GetXCApiNames(int timeout = 10000)
+        {
+            throw new NotImplementedException("Method not available for RabbitMq implementation");

```

Comment:
Only the WebSocket bridge provides xcApi lists and files. The message should be re-formatted though.
_"Not supported for Rabbit MQ"_ maybe.